### PR TITLE
Dev.named tuple

### PR DESCRIPTION
**Pull request by dlothian**
Compatible with the new data format for the "output" attribute of a conjugation ResponseObject.

### CODE REVIEW

File: projects/word-weaver/src/app/core/conjugation/conjugation.service.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { selectSettingsState } from "../core.state";
 import { SettingsState } from "../settings/settings.model";
 import { TableviewerState } from "../tableviewer-selection/tableviewer-selection.model";
 import { WordmakerState } from "../wordmaker-selection/wordmaker-selection.model";
+import { type } from "os";
```
Comment (roedoejet) [inline projects/word-weaver/src/app/core/conjugation/conjugation.service.ts]: This import is errant - please remove.

File: projects/word-weaver/src/app/core/conjugation/conjugation.service.ts [hunk 2]
```diff
@@ -66,6 +67,7 @@ export class ConjugationService {
     const filterValues = { root: [], option: [], agent: [], patient: [] };
     ["option", "agent", "patient", "root"].forEach(x => {
       if (selection[x]) {
+        console.log(x, selection[x]);
```
Comment (roedoejet) [inline projects/word-weaver/src/app/core/conjugation/conjugation.service.ts]: We don't need console.logs in here in production

**Review by roedoejet — CHANGES_REQUESTED** [review]
Just looks like the `projects/word-weaver/src/app/core/conjugation/conjugation.service.ts` file maybe got accidentally committed. After removing that it all looks good to me, thanks @dlothian !

**Comment by joanise** [comment]
These changes need to be made in parallel in `config.fr.ts` and wordweaver's backend `fr.model.py`. Working on that now.

**Review by joanise — APPROVED** [review]
This is ready to merge now.

**Review by roedoejet — APPROVED** [review]
great thanks all!

**Pull request merged**